tests/test_backends_torch_gradients.py

```diff
@@ -0,0 +1,89 @@
+import sys
+
+import numpy as np
+import pytest
+
+from qibo import gates, models
+from qibo.quantum_info import infidelity
+
+
+def test_torch_gradients(backend):
+    if backend.name != "pytorch":
+        pytest.skip("Test only valid for PyTorch backend.")
```

Comment:
```suggestion
def test_torch_gradients():
    backend = PyTorchBackend()
```
no need to use the fixture if only `pytorch` is tested